sdk/python/lib/setup.py

```diff
@@ -23,8 +23,8 @@
       license='Apache 2.0',
       packages=find_packages(),
       install_requires=[
-          'google==2.0.1',
```

Comment:
Turns out the `google` package is a random library that scrapes the Google frontpage and lets you do programmatic searches. The dependency we actually want is `protobuf`, which provides the `google.protobuf` module.